compiler/tflchef/core/src/ModelChef.cpp

```diff
@@ -219,6 +189,20 @@ struct CookParams
   std::vector<std::string> &custom_code_vec;
   std::string noname;
 };
+#endif
+
+struct ModelChef
```

Comment:
`ModelChef` is introduce to replace `CookParams` and later to provide huge Buffer management.